src/pages/documentation/policy/license.astro

```diff
@@ -84,24 +84,16 @@ const page: IStandardPageMetadata = {
       last updated date if it is available in the metadata or the date you accessed/downloaded the data if not. For example,
       this is what it could look like in an MLA-esque format:
     </p>
-    <Blockquote>
-      <ul>
-        <li>
-          State Geographic Information Database. <em>Utah Address Points</em>. (Feature class). Salt Lake City, UT: Utah
-          Geospatial Resource Center, July 2021. Geospatial point data. 27 July 2021.
-          https://gis.utah.gov/data/location/address-data/.
-        </li>
-      </ul>
-    </Blockquote>
+    <MonoQuote>
+      State Geographic Information Database. <em>Utah Address Points</em>. (Feature class). Salt Lake City, UT: Utah
+      Geospatial Resource Center, July 2021. Geospatial point data. 27 July 2021.
+      https://gis.utah.gov/data/location/address-data/.
```

Comment:
**polish**: with long continues "words" like this, caution needs to be taken since they will overflow on mobile screens. the `<wbr/>` tag is our friend here.

Reply:
TIL, cool.